quizzle-book/categories/style.css

```diff
@@ -0,0 +1,86 @@
+@import url('https://fonts.googleapis.com/css2?family=Baloo+2:wght@700&display=swap');
+@import url("../../css/colors.css");
+
+body {
+    background: var(--secondary);
+    font-family: 'Baloo 2', cursive;
+    color: var(--smoke);
+}
+
+.nav-header {
+    background: var(--primary-color);
+    font-size: 1.5rem;
+    padding: 0 4rem;
+}
+
+.menu-list-items {
+    font-size: 1.2rem;
+    padding: 0.8rem 1rem;
+}
+
+.menu-list-items a {
+    color: var(--smoke);
+    text-decoration: none;
+}
+
+.menu-list-items:hover {
+    background: var(--primary-color);
+    filter: brightness(90%);
+}
+
+.container {
+    display: flex;
+    justify-content: center;
+    padding: 2rem;
+    gap: 2rem;
+    flex-wrap: wrap;
+}
+
+.card {
+    width: 20rem;
+    backface-visibility: hidden;
+}
+
+.flip-card {
+    position: relative;
+    transform-style: preserve-3d;
+    transform: rotateY(0deg);
+    transition: all 0.5s ease-in-out;
+}
+
+.card .content-desc, .card .btn {
+    font-size: 1.2rem;
+}
+
+.btn-grp {
+    margin: 1rem;
+    margin-top: 0;
+    flex-direction: row-reverse;
```

Comment:
Maybe avoid reordering here if you can change the order in the HTML document.  As it's really not good for accessibility.
https://web.dev/learn/css/flexbox/#reversing-the-flow-of-items-and-accessibility

Reply:
Oh thanks, I forgot that if my properties are getting reversed then the key functions will do it too😂